docker-nginx/templates/default.conf.template

```diff
@@ -173,7 +194,6 @@ server {
     proxy_intercept_errors on;
 
     proxy_pass $redirect_uri;
-
```

Comment:
I guess this change was by accident?

Avoid random changes like this to reduce the unneccesary changes. Now it is fine, but especially if it is the only change in the PR it makes later git greping a bit of a mess.